pages/linux/auracle.md

```diff
@@ -0,0 +1,20 @@
+# auracle
+
+> Auracle is a command line tool used to interact with Arch Linux's User Repository, commonly referred to as the AUR.
+> More information: <https://github.com/falconindy/auracle>.
+
+- Find packages in the AUR by regular expression:
```

Comment:
Maybe this is better:
```suggestion
- Display AUR packages that match a regular expression:
```

Reply:
Reasonable, but just stolen from: https://github.com/falconindy/auracle#features - we search/find packages, right?